src/tests/gssapi/t_channel_bindings.c

```diff
@@ -0,0 +1,113 @@
+/* -*- mode: c; c-basic-offset: 4; indent-tabs-mode: nil -*- */
+/*
+ * Copyright 2020 by the Massachusetts Institute of Technology.
```

Comment:
Is this the right copyright?  It's not what we (Red Hat) use for this project.

Reply:
That's the old MIT copyright, probably copied from another source file.  We now use the one in src/prototype/prototype.c.